pkg/morph/client/notary.go

```diff
@@ -769,7 +769,12 @@ func (c *Client) CalculateNonceAndVUB(hash util.Uint256) (nonce uint32, vub uint
 
 	height, err := c.getTransactionHeight(conn, hash)
 	if err != nil {
-		return 0, 0, fmt.Errorf("could not get transaction height: %w", err)
+		c.logger.Warn("could not get transaction height", zap.Error(err))
+		vub, err = c.notaryTxValidationLimit(conn)
```

Comment:
Tricky one when we're close to the epoch boundary, but we don't have a lot of choices currently.

Reply:
what do you mean? it seems like an epochless algorithm

Reply:
Yes, ok.